clients/src/main/java/org/apache/kafka/common/requests/OffsetFetchRequest.java

```diff
@@ -51,86 +49,9 @@ public static class Builder extends AbstractRequest.Builder<OffsetFetchRequest>
         public final OffsetFetchRequestData data;
```

Comment:
Given that you are refactoring this file, could you please change the modifier from public to private? there is already a method `data()`